app/javascript/views/userDetail/ChangeLog.jsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react'
+import { Rolodex, Card, CardBody, CardHeader, CardTitle, DataGrid } from '@cwds/components'
+
+const columnConfig = [
+  {
+    Header: 'Date/Time',
+    accessor: 'time_stamp',
+    minWidth: 75,
+  },
+  {
+    Header: 'Type',
+    accessor: 'type',
+    minWidth: 70,
+  },
+  {
+    Header: 'Made By',
+    accessor: 'admin_name',
+    minWidth: 70,
+  },
+  {
+    Header: 'Notes & Details',
+    accessor: 'notes',
+    minWidth: 50,
+  },
+]
+
+const data = [
+  { time_stamp: 'Thu Mar 03 2018 14:22:43', type: 'Permission Changes', admin_name: 'H' },
```

Comment:
We need to think about this time_stamp value.  It doesn't sort correctly, even though this is fake data.  Is it the format we'll receive it in?

Reply:
I don't think so and I am not sure in which format will receive the time-stamp. I just represented this way since we are formatting other time related data-fields to show in this format on the list and details page. That may change based on the design in next iteration.

Reply:
I think when the actual API works we can assume it's going to have format like " "last_login_date_time": "2019-01-08 14:52:49". and that we should use DataGrid (react-table's) accessor / id style sorting mechanism.  See UserList.jsx and 

```suggestion
  const formatDate = row => {
  return checkDate(row.time_stamp)
}

...

const columnConfig = [
  {
    Header: 'Date/Time',
    id: 'time_stamp',
    accessor: formatDate,
    minWidth: 75,
  },
  
  ...
```